lib/active_merchant/billing/gateways/bridge_pay.rb

```diff
@@ -114,7 +114,7 @@ def add_payment_method(post, payment_method)
           post[:TransitNum] = payment_method.routing_number
           post[:AccountNum] = payment_method.account_number
           post[:NameOnCheck] = payment_method.name
-          post[:ExtData] = "<AccountType>#{payment_method.account_type.capitalize}</AccountType>"
+          post[:ExtData] = "<AccountType>#{payment_method.account_type[0].capitalize}</AccountType>" if payment_method.account_type
```

Comment:
I don't believe the `account_type` property is an array. Is the `[0]` necessary here?

Reply:
Duh.. this is Ruby - it could also work for strings.

Reply:
How about adding a unit test for this fix?

Reply:
Gah, I did, but forgot to `git add -p` as I was bouncing between two other branches. Will grab it out of whatever stash it landed in and add.

And re. the array access: AFAICT, the original implementation was wrong. In AM, if I understand properly, `account_type` should be one of `checking` or `savings`, but Bridge Pay actually requires the value `C` or `S` (see https://www.dropbox.com/s/1tg479a9n8e3dvt/BridgeComm%20Communication%20Interface%20API%20Guide.pdf?dl=0 page 96, which I think is what should be happening here).